src/main/java/DBAccesser/User/QueryUser.java

```diff
@@ -0,0 +1,57 @@
+package DBAccesser.User;
+
+import Entity.Class.User;
+
+import org.springframework.stereotype.Component;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.gcp.data.spanner.core.SpannerTemplate;
+
+import com.google.cloud.spanner.Statement;
+import org.springframework.cloud.gcp.data.spanner.core.SpannerQueryOptions;
+
+import java.util.List;
+
+@Component
+public class QueryUser {
+    
+    @Autowired
+    SpannerTemplate spannerTemplate;
+
+    public boolean checkIfUserExists(String username, String emailID) {
+
```

Comment:
Nit: Remove such blank lines in beginning of functions. They just eat up vertical space without adding any clarity.